### DISCUSSION TITLE
How can I backup with yetiforce inc database

### DISCUSSION THREAD

**Discussion by MonsterITServices** [YetiForceCompany/YetiForceCRM#16805] [How to] [answered]
Hi,
I've read through the backup section in yetiforce's website but how do i backup the site?

I've got the cron working but there is no backup's showing on the system and no errors. So would like to know how to initiate a backup please?

**Comment by s2b-git** [comment, marked as answer]
snapshot, system backup, rsync, dump, filezilla etc. ;-)
